api/v08/kogitoserverlessworkflow_types.go

```diff
@@ -225,11 +236,11 @@ type FunctionRef struct {
 	RefName      string            `json:"refName"`
 	Arguments    map[string]string `json:"arguments,omitempty"`
 	SelectionSet *string           `json:"selectionSet,omitempty"`
-	Invoke       *InvokeTye        `json:"invoke,omitempty"`
+	Invoke       *InvokeType       `json:"invoke,omitempty"`
 }
 
 type Action struct {
-	Name               *string           `json:"name"`
+	Name               *string           `json:"name,omitempty"`
```

Comment:
on v08 name can be null, no need to be a pointer.

Reply:
@spolti Thanks for letting me know, i will change it during my next commit.